src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1001,7 +1001,7 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
         if ($config['version_strategy']) {
             $defaultVersion = new Reference($config['version_strategy']);
         } else {
-            $defaultVersion = $this->createVersion($container, $config['version'], $config['version_format'], $config['json_manifest_path'], '_default');
+            $defaultVersion = $this->createVersion($container, $config['version'], $config['version_format'], $config['json']['manifest_path'], '_default', $config['json']['manifest_allow_missing']);
```

Comment:
For backwards compatibility we still need to make sure that we handle the legacy setting. We can do that in the `Configuration` class and map the old to the new option there.

Reply:
I'm on this, not that easy it seems.

Reply:
done